custom_reports/http_api/paths/query.yaml

```diff
@@ -0,0 +1,98 @@
+post:
+  tags:
+  - querying
+  summary:
+    Execute query
+  description: |
+    <p>API that allows querying the database directly. Please note that each
```

Comment:
Formatting of text in OpenAPI is tricky singe we use 2 plugins to render it and both of them render it using different parser (Redoc - Markdown, OpenAPI - Sphinx).

HTML will be passed by makdown, but Sphinx will escape it. Try using formatting that works in both if you can.
(e.g. http://docutils.sourceforge.net/docs/user/rst/quickref.html#paragraphs)

Reply:
removed html